### PR TITLE
Some basic UI; start on module structure

### PR DESCRIPTION
**Pull request by hbarudin**
Putting in a PR on this just because I think it's hit critical mass of code review, especially since it's almost entirely front-end stuff.

I also wanted to stop here because I want to talk with you @williswoodward about what can be represented in the Rails DB as a model (the information about a module; maybe there can be a 'module' model?) since I was about to represent it all on the front end but that doesn't take advantage of rails at all! Anyway, I'm babbling.

This is based on @williswoodward 's last PR ( https://github.com/hbarudin/songatfirstsight/pull/4 ), so only the last 4 commits here are my new code. Once that PR gets merged I will rebase this so it's purely my code showing in the diff.

**tl;dr** let's talk about what Rails models we might want (or at least a few of them)

### CODE REVIEW

**Comment by williswoodward** [comment]
I have been SUPER sick this week but am on the mend and will look at this
tonight!!

**Comment by williswoodward** [comment]
This looks good to me - basic structural stuff, good place to start.

For sure, we should have a back-and-forth-y conversation about what we want to store in the database. I will have TONS of time next week (starting Monday night), let's schedule a call!
